src/Faker/Provider/ru_RU/Person.php

```diff
@@ -154,4 +154,23 @@ public function middleName($gender = null)
             static::GENDER_FEMALE,
         )));
     }
+
+    /**
+     * Return last name for the specified gender.
+     *
+     * @access public
```

Comment:
Not sure if using the `@access` annotation here is something we really need - it duplicates the `public` visibility keyword that is already present. Can we remove it?